src/components/input-and-preview.less

```diff
@@ -17,6 +17,6 @@
   background-color: @edu-background-color-light;
   padding: 4px 11px;
   display: grid;
-  min-height: @edu-control-height-base;
+  min-height: 2 * @edu-control-height-base;
```

Comment:
This will probably break the inline preview (which is a one-line input), although we do not have any of those on the page.

Reply:
Good point, fixed with passing `inline` prop and adjusting min height accordingly.